src/api/operator/types.ts

```diff
@@ -63,10 +63,7 @@ export type Trade = Pick<RawTrade, 'blockNumber' | 'logIndex' | 'owner' | 'txHas
   orderId: string
   kind?: OrderKind
   buyAmount: BigNumber
-  executedBuyAmount?: BigNumber
   sellAmount: BigNumber
-  executedSellAmount?: BigNumber
-  executedFeeAmount?: BigNumber
```

Comment:
This was never necessary as they don't exist in the original Trade coming from the API and have equivalent non `executed` counterparts.